File: Core Java/accountPolymorphism/src/com/techlabs/savingAccount/SavingAccount.java
package com.techlabs.savingAccount;

import com.techlabs.account.Account;

public class SavingAccount extends Account {

	public SavingAccount(int account, String name, double balance) {
		super(account, name, balance);
	}

	@Override
	public void withdraw(double balance) {

		if (this.balance >= 500 && this.balance - balance >= 500) {
			this.balance = this.balance - balance;
		} else {
			throw new RuntimeException(
					"Your balance is low Minimum balance require 500");
		}
	}

}

File: DOTNET/C#/AccountPolymorphism/AccountPolymorphism/SavingAccount.cs
using System;

namespace AccountPolymorphism
{
    class SavingAccount : Account
    {
        public SavingAccount(int accountNumber, string name, double balance) : base(accountNumber, name, balance)
        {
        }

        public override void WithDraw(double balance)
        {
            if (this.balance >= 500 && this.balance - balance > 500)
            {
                this.balance = this.balance - balance;
            }
            else
            {
                throw new Exception("Your Balance is low.Minimum balance require is 500..");
            }
        }
    }
}
